integration-tests/bats/sql-local-remote.bats

```diff
@@ -438,4 +438,20 @@ get_staged_tables() {
     run dolt sql -q "show tables"
     [ "$status" -eq 1 ]
     [[ "$output" =~ "When a password is provided, a user must also be provided" ]] || false
+
+@test "sql-local-remote: verify simple dolt reset behavior" {
+    start_sql_server altDB
+    dolt --verbose-engine-setup --user dolt sql -q "create table test1 (pk int primary key)"
```

Comment:
I suspect these won't work after my authentication change that you just rebased onto. You should actually just drop the `--user` flag from all inputs to new tests. It doesn't add anything to the tests.